sdcm/sct_config.py

```diff
@@ -429,6 +429,12 @@ class SCTConfiguration(dict):
         dict(name="aws_root_disk_name_monitor", env="SCT_AWS_ROOT_DISK_NAME_MONITOR", type=str,
              help=""),
 
+        dict(name="aws_root_disk_size_loader", env="SCT_AWS_ROOT_DISK_SIZE_LOADER", type=int,
+             help=""),
```

Comment:
please, add a precise description of this new attribute inside help paramater

Reply:
what's not self explanatory in the parameter name here? all the similar params have no help text.